tests/browser/heda_settings_affiliations_edit_mode.robot

```diff
@@ -15,17 +15,16 @@ Validate Edit Mode For Affiliations, Settings
     #Set focus to the status field
     Click Element                   //div[@class='slds-button-group']//span[contains(text(), 'Edit')]
     Wait Until Element Is visible   //div[@class='slds-button-group']//span[contains(text(), 'Save')]
-    
-    #Enter known value into status field, then save the form
-    Click Element                   //div[@class='slds-tabs--scoped']//input[@type='radio']
-    Input text                      //div[@class='slds-tabs--scoped']//input[@type='text']      StatusTest1
     Textfield Value Should Be       //div[@class='slds-tabs--scoped']//input[@type='text']      StatusTest1
 
+    #Save the changes
+    Select Checkbox                 //label[@class='slds-checkbox']//input[@type='checkbox']
 
-#    #Save the changes
-#    Select Checkbox                 //label[@class='slds-checkbox']//input[@type='checkbox']
-#    Click Element                   //div[@class='slds-button-group']//span[contains(text(), 'Save')]
-    
+    # Record Type Validation
+    Click Element                   //input[contains(@class,'store-errors')]/parent::label
```

Comment:
@dashattuck curious to hear what this line does. Does this line or "Select Checkbox" check the checkbox?

Reply:
This places a check in the checkbox for Enable Record Type Validation.  It subsequently gets saved.

Reply:
@dashattuck yes that makes sense. I am just a little surprised to see it is finding the "store-errors" class.

Reply:
@dashattuck feel free to change the class name reference here.